### PR TITLE
adding check on /github/activity endpoint for webhook secret validation,

### PR DESCRIPTION
**Pull request by makkalot**
addresses #1870 
- load the github webhook secret from ssm
- check for the signature of the request on `/github/activity`

### CODE REVIEW

**Comment by makkalot** [comment]
@dealako this pr needs `GITHUB_APP_WEBHOOK_SECRET` set before being deployed!

File: cla-backend/cla/controllers/github.py [hunk 1]
```diff
@@ -759,21 +759,68 @@ def validate_organization(body):
...
+    event type: {event_type},
+    user login: {user_login},
+    repository_id: {repository_id}, repository_name : {repository_name},
+    installation_id: {installation_id}"""
 
-    return True if hmac.compare_digest(mac.hexdigest(), signature) else False
+    body = f"""
+    <p>Hello EasyCLA Maintainer,</p>
+    <p>This is a notification email from EasyCLA regarding failure of webhook secret validation.</p>
+    <p>{msg}</p>"""
```
Comment (dealako) [inline cla-backend/cla/controllers/github.py:804]: Very good, but we want to add the standard help doc/links and the sign-off line to the end of the system generated emails.

**Review by dealako — CHANGES_REQUESTED** [review]
Minor updates requested.

File: cla-backend/cla/config.py [hunk 2]
```diff
@@ -68,7 +68,7 @@
 GITHUB_PR_NOTIFICATION = 'status+comment'
 
 # GitHub Application Service.
-GITHUB_APP_WEBHOOK_SECRET = 'webhook-secret'
+GITHUB_APP_WEBHOOK_SECRET = os.getenv("GITHUB_APP_WEBHOOK_SECRET", "")
```
Comment (dealako) [inline cla-backend/cla/config.py:71]: Can we load this via AWS SSM like the other params?

File: cla-backend/cla/config.py [hunk 3]
```diff
@@ -77,6 +77,9 @@
 EMAIL_SERVICE = 'SNS'  #: Email service to use for notification emails.
 EMAIL_ON_SIGNATURE_APPROVED = True  #: Whether to email the user when signature has been approved.
 
+# Platform Maintainers
+PLATFORM_MAINTAINERS = ["ddeal@linuxfoundation.org", "makkalot@gmail.com"]
```
Comment (dealako) [inline cla-backend/cla/config.py:81]: Can we load this via AWS SSM like the other params?

File: cla-backend/cla/config.py [hunk 4]
```diff
@@ -77,6 +77,9 @@
...
+
 # SMTP Configuration.
 #: Sender email address for SMTP service (from address).
 SMTP_SENDER_EMAIL_ADDRESS = 'test@cla.system'
```
Comment (dealako) [inline cla-backend/cla/config.py:85]: I know this wasn't part of your issue/PR, but can we also load this from AWS SSM? We may need to create a new parameter.

**Review by dealako — CHANGES_REQUESTED** [review]

**Comment by dealako** [comment]
@makkalot please review the above comments. Overall, very good. Let's externalize these parameters via AWS SSM.

File: cla-backend/cla/config.py (hunk 2 above)
Comment (makkalot) [reply cla-backend/cla/config.py:71]: this is actually how we pass from AWS SSM in my understanding(for Python side), we read them from `serversless.yaml` and then they're passed as env variables to the process.

File: cla-backend/cla/config.py (hunk 3 above)
Comment (makkalot) [reply cla-backend/cla/config.py:81]: sure 👍

File: cla-backend/cla/config.py (hunk 4 above)
Comment (makkalot) [reply cla-backend/cla/config.py:85]: sure 👍

File: cla-backend/cla/controllers/github.py (hunk 1 above)
Comment (makkalot) [reply cla-backend/cla/controllers/github.py:804]: got it 👍

**Comment by makkalot** [comment]
This PR would need 3 ssm key set before being deployed : 

- SMTP_SENDER_EMAIL_ADDRESS : `ssm:/cla-smtp-sender-email-address-${opt:stage}`
- PLATFORM_MAINTAINERS : list of emails  : `ssm:/cla-lf-platform-maintainers-${opt:stage}`
- GITHUB_APP_WEBHOOK_SECRET: `ssm:/cla-gh-app-webhook-secret-${opt:stage}`

**Review by dealako — APPROVED** [review]
👍

**Pull request merged**